easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'fastai'
+version = '2.7.10'
+
+homepage = 'https://www.fast.ai/'
+description = """The fastai deep learning library."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['ccef6a185ae3a637efc9bcd9fea8e48b75f454d0ebad3b6df426f22fae20039d']
+
+download_dep_fail = True
+use_pip = True
+
+builddependencies = [('binutils', '2.38')]
+multi_deps = {'Python': ['3.10.4']}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
+}
```

Comment:
this is already the default sanity check, so can be removed.

Reply:
Done!